shell/platform/tizen/BUILD.gn

```diff
@@ -235,36 +237,26 @@ template("embedder") {
 
 embedder("flutter_tizen_mobile") {
   target_type = "shared_library"
-  use_evas_gl_renderer = false
-
```

Comment:
nit: Please leave the blank line between `target_type` and `defines` as it is since they're not related to each other.

Reply:
done